test/functional/apps/context/_discover_navigation.js

```diff
@@ -77,5 +77,22 @@ export default function({ getService, getPageObjects }) {
 
       expect(hasDisabledFilters).to.be(true);
     });
+
+    it('should go back after loading', async function() {
+      // get the timestamp of the first row
+
+      // navigate to the context view
+      await docTable.clickRowToggle({ rowIndex: 0 });
+      await (await docTable.getRowActions({ rowIndex: 0 }))[0].click();
+      await PageObjects.context.waitUntilContextLoadingHasFinished();
```

Comment:
I think this fails, because in the test before, you're in the context / surrounding docs view. 
doesn't fail, if you just execute this one. Ideally this tests should work independently, but this would mean that the whole file has to be refactored